Jenkinsfile

```diff
@@ -287,10 +287,12 @@ def get_system_tests_pipeline() {
             sh """cd system-tests/
             scl enable rh-python35 -- python -m pytest -s --junitxml=./SystemTestsOutput.xml .
             """
-            junit "system-tests/SystemTestsOutput.xml"
-            archiveArtifacts "system-tests/logs/*.log"
           }  // stage
         } finally {
+          stage("System tests: Archive") {
+            junit "system-tests/SystemTestsOutput.xml"
```

Comment:
As this is inside the `finally` block, it might be desirable to make this step fail-proof in the case test files are not created, to prevent the clean-up stage from being skipped. This can be done by changing this line to `junit allowEmptyResults: true, testResults: 'system-tests/SystemTestsOutput.xml'`. It has one disadvantage, though, as if the tests do not run because of a Jenkins job configuration issue (in which case files would not be generated), you might miss it. A simple workaround would be to move the clean-up stage to the start of the `finally` block.